src/ripple/app/tx/impl/apply.cpp

```diff
@@ -55,10 +55,82 @@ invoke_preflight (PreflightContext const& ctx)
     case ttAMENDMENT:
     case ttFEE:             return Change           ::preflight(ctx);
     default:
+        assert(false);
         return temUNKNOWN;
     }
 }
 
+/*
+    invoke_preclaim<Tx> uses name hiding to accomplish
+    compile-time polymorphism of (presumably) static
+    class functions for Transactor and derived classes.
```

Comment:
It's not immediately obvious, though, and it was enough to trip up @scottschurr at first (https://github.com/ximinez/rippled/commit/5404e614a895bdaeee5e60a2bef7aca7f647d8a1#commitcomment-13477808). If you're just looking casually, the comment saves the time needed to figure that out.